[    3.1056][main][error] Column names should be valid identifiers.
Traceback (most recent call last):
  File "./cubes.py", line 89, in <module>
    main()
  File "./cubes.py", line 62, in main
    specification = Specification(spec)
  File "/home/rmbb/CUBES/squares/dsl/specification.py", line 108, in __init__
    df = read_table(input)
  File "/home/rmbb/CUBES/squares/dsl/specification.py", line 39, in read_table
    raise ValueError(f'Columns names should be valid identifiers. Column {col} in table {path}.')
ValueError: Columns names should be valid identifiers. Column pre-Ba in table tests-examples/scythe/recent_posts/tables/029_1.csv.
